github/store/db.go

```diff
@@ -47,72 +47,180 @@ const (
 )
 
 var tables = []string{
-	"organizations_versioned",
-	"users_versioned",
-	"repositories_versioned",
-	"issues_versioned",
-	"issue_comments_versioned",
-	"pull_requests_versioned",
-	"pull_request_reviews_versioned",
-	"pull_request_comments_versioned",
+	"github_organizations_versioned",
+	"github_users_versioned",
+	"github_repositories_versioned",
+	"github_issues_versioned",
+	"github_issue_comments_versioned",
+	"github_pull_requests_versioned",
+	"github_pull_request_reviews_versioned",
+	"github_pull_request_comments_versioned",
 }
 
 func (s *DB) SetActiveVersion(ctx context.Context, v int) error {
-	// TODO: for some reason the normal parameter interpolation $1 fails with
-	// pq: got 1 parameters but the statement requires 0
+	// Unified schema
 
-	_, err := s.DB.ExecContext(ctx, fmt.Sprintf(`CREATE OR REPLACE VIEW organizations AS
+	_, err := s.DB.ExecContext(ctx, "DROP MATERIALIZED VIEW IF EXISTS owners")
```

Comment:
Premature optimization is the root of all evil, but could it be that using https://www.postgresql.org/docs/10/sql-refreshmaterializedview.html could be beneficial in terms of performances?

Reply:
Also once we have multiple providers we need to think about this, as we cannot make each provider's storer run these independently. We would end up with only the data of the last provider that finished if I didn't miss anything. I guess that we would need a single _db committer_.

Reply:
I added drop here to be able to develop first of all. Also, we might want to keep it for the first (maybe unstable) versions to allow changing the views without migration. We will have to improve it at some point but I don't think we should do it right now.

> single db committer

at the time of writing this code there was single commiter. It changed only in master. I'll fix it during rebase of course.

Reply:
oh. maybe you mean distributed mode here, not multiple downloaders that we use in our cli.
yes, there is no support for distributed mode yet. imo better implement it in a separate pr later.

Reply:
yup, I meant multiple downloaders in terms of multiple storers. If we implement a similar thing for BitBucket, for example, we will have a `storer` of a downloader for the first provider that would run `SetCurrent` which drops the materialized views and puts data, then a `storer` of a downloader for the second provider tha would do the same. So, in the end, we will have only data for the second provider, unless I missed something.

But for sure not something to be addressed now. Just a note for the nex things to do to keep in mind. 👍

Reply:
> we will have only data for the second provider

As we agreed in the DD, for now, we support only 1 provider.

For the problem with distributed mode I have created an issue: https://github.com/src-d/metadata-retrieval/issues/86